nailgun/entities.py

```diff
@@ -844,9 +844,11 @@ def __init__(self, server_config=None, **kwargs):
             'command': entity_fields.StringField(
                 required=True,
                 str_type='latin1',
+                default='top',
```

Comment:
1) Actually, I am performing all development using CentOS default installation image and do nothing extra to have 'top' command, but I have it.
2) We can use another command that any Linux package has. For example 'ls' (as far as I know).
3) I like to have that statement, because that minimize code from robottelo side and anyway you will provide some value for that field, so it doesn't matter - you have failure on Nailgun or Robottelo side

But I am fine with any solution, just let me know your decision, because I have PR in Robottelo blocked by this one